### DISCUSSION TITLE
Automation upon triggered smoke detectors

### DISCUSSION THREAD

**Discussion by gymoese** [ptz0n/homebridge-verisure#99] [Q&A]
Hi,

It is possible to let the smoke detectors appear as real detectors, and thereby for example start an automation if the alarm is triggered?

Regards

Peter

**Comment by ptz0n** [comment]
Hi Peter & thanks for reaching out. The current implementation only supports temperature and humidity readings.

This have been discussed before and we came to the conclusion that it's not a great idea since we are polling for the current/last values. We need to increase the polling frequency quite a bit to act quickly if there is smoke detected. Verisure isn't happy about polling for values too frequently.

Can you tell us more about your use case?

**Reply by gymoese** [reply]
Hi Erik, it makes sense, but it would however be really cool to be able to turn on all my hue lights during an alarm. I have come up with a couple of ideas I will try out. I have a number of Verisure smart plugs in spare. I can create an automation in the Verisure app, which turns on the smart plug upon an alarm. Then I can make a HomeKit automation reacting on the Verisure smart plug being turned on, and thereby trigger my ligth automation. Another, and most likely faster solution would be to use the Verisure smart plug to turn another smart device, like a bulb or perhaps a Hue smart plug, and then use this event to trigger the HomeKit automation - that should be as fast as what Verisure can deliver ;-) Anyway, thanks for the answer.

**Comment by ptz0n** [comment]
Yes, using the smartplugs as a trigger is a great idea. That should work! Please share your experiences once you have it all configured.

**Reply by gymoese** [reply]
I seems to have a problem using the smart plugs as trigger, as their state is not updated unless I open the Home app or another app needing the state. 

I can follow this behaviour in the logs on homebridge. All my door sensors, are updated regularly and so is the alarm state, but not the state of the smart plugs or the smoke detectors - which makes ok sense, unless you want to use them as triggers for something :-)

It would be great to be able to select a set of "passive" components like smart plugs to be updated regularly, making triggers possible.

**Reply by ptz0n** [reply]
Oh, yes. These have the same polling interval as smoke detectors. I thought you were going use a smart plug to control a different ecosystem, like Hue. That way you will get a much faster time to event, if you think 30 seconds (max) is a bit slow.
